libs/ui/src/app_base.test.tsx

```diff
@@ -98,3 +98,19 @@ test('renders with selected themes', () => {
     parseCssColor(expectedTheme.colors.foreground)
   );
 });
+
+test('renders with enableScroll', () => {
+  const { container } = render(
+    <AppBase enableScroll>
+      <div>foo</div>
+    </AppBase>
+  );
+
+  expect(container).toContainHTML('<div>foo</div>');
+
+  const htmlNode = document.body.parentElement;
+  assert(htmlNode);
+  const computedStyles = window.getComputedStyle(htmlNode);
+
+  expect(computedStyles.overflow).toEqual('auto');
+});
```

Comment:
Other than meeting coverage, what is the point of this test? This is the kind of test I'm uncertain about, in terms of value.

Reply:
I suppose in this case, we're confirming that a property is being passed to to global styles. That actually seems worth it to me. So I would take it slightly out of the category of tests that just confirm that passing in "green" makes the styling "green".

Reply:
TBH, was originally driven by meeting coverage, but agreed that this sort of falls in the functional category